rapid_response_xblock/models.py

```diff
@@ -0,0 +1,42 @@
+"""
+Models for storing rapid response grades
+"""
+from __future__ import unicode_literals
+
+from django.conf import settings
+from django.db import models
+from django.utils.encoding import python_2_unicode_compatible
+from jsonfield import JSONField
+from model_utils.models import TimeStampedModel
+from opaque_keys.edx.django.models import (
+    CourseKeyField,
+    UsageKeyField,
+)
+
+
+@python_2_unicode_compatible
```

Comment:
Is this something we need to decorate all of our models with if they store course/block keys?

Reply:
The decorator only handles the `__str__` function: https://docs.djangoproject.com/en/1.11/ref/utils/#django.utils.encoding.python_2_unicode_compatible